lib/error.js

```diff
@@ -1,6 +1,21 @@
-const { assoc, ifElse, is, merge, pick, pipe, prop, when } = require('ramda')
+const {
+  assoc,
+  compose,
+  converge,
+  ifElse,
+  is,
+  lensProp,
+  merge,
+  objOf,
+  over,
+  pick,
+  pipe,
+  prop,
+  when,
+} = require('ramda')
```

Comment:
You've probably got an editor plugin that does this reformatting for you automatically, but it's best to follow the coding style of the project you're working on.

Reply:
You're right, of course. I couldn't find a precedent for a long destructured `require` statement.